spec/twilioruby_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilioruby'
```

Comment:
Amazing that you managed to write rspec for the 'text sms'. I didn't know how where to start with this one.